src/app/lib/analyticsUtils/index.js

```diff
@@ -261,3 +261,7 @@ export const getComponentInfo = ({ result, componentName, componentData }) => {
     },
   };
 };
+
+export const getLibraryVersion = () => {
```

Comment:
If all this function does is return 'simorgh' string why can't we just make it a string constant called `LIBRARY_VERSION` or so?

Reply:
@samora This has been refactored